addons/web/static/tests/helpers/qunit_config.js

```diff
@@ -131,6 +131,9 @@ QUnit.on('OdooAfterTestHook', function () {
             }
 
             if (!isValid) {
+                console.log('Body still contains undesirable elements:' +
```

Comment:
If you use console.error instead of  console.log, the error can be catched by the python code that drives the Chrome browser. That way, the error appears on the build page in the runbot.
https://github.com/odoo/odoo/blob/bf1e39540f53c3180eca56a5adc7b30d3334309b/odoo/tests/common.py#L746-L748
I'm not sure if it's the best option but it could help to quickly catch the error

Reply:
Ok sounds good.